cli/internal/fs/turbo_json.go

```diff
@@ -301,6 +300,54 @@ func (c *TaskDefinition) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// MarshalJSON deserializes JSON into a TaskDefinition
+func (c *TaskDefinition) MarshalJSON() ([]byte, error) {
+	// Initialize with empty arrays, so we get empty arrays serialized into JSON
```

Comment:
For elegance I _kinda_ want to omit empty arrays when printing, but that may make it harder for people to use `--dry=json`. So I'm not actually proposing this.

Reply:
Yeah, IMO it's actually important to print the empty arrays, so it's never a question about what the whole config is. In other words, missing key can mean "we don't have this config" or "this config has an empty value", and that makes debugging a bit harder.